testdata/terraform/cloudflare_user_agent_blocking_rule/test.tf

```diff
@@ -0,0 +1,10 @@
+resource "cloudflare_user_agent_blocking_rule" "terraform_managed_resource" {
+  description = "My description 1"
+  mode        = "whitelist"
```

Comment:
`mode = "whitelist"` isn't possible anymore. we should swap this to match the available API parameters to test.

Reply:
Oh, that's a bit puzzling, actually! Here in cloudflare-go: https://github.com/cloudflare/cloudflare-go/blob/master/user_agent.go#L48, the supported ones are “block”, “challenge”, “js_challenge”, “whitelist”. In the public API docs, instead of “whitelist” there's “managed_challenge”. However, when I try to POST to the API, it accepts all. Just double-checked Kraken, and it validates against all 5 modes.

Reply:
I have updated it to `js_challenge` nevertheless.